assembl/static2/js/app/client.js

```diff
@@ -27,6 +29,31 @@ const myFragmentMatcher = new IntrospectionFragmentMatcher({
 
 const dataIdFromObject = o => o.id;
 
+const networkInterface = createNetworkInterface({
+  uri: getFullPath('graphql', { slug: getDiscussionSlug() }),
+  opts: {
+    credentials: 'same-origin'
+  }
+});
+
+// trace every graphql operation in sentry
+networkInterface.use([
+  {
+    applyMiddleware: function (req, next) {
+      Sentry.addBreadcrumb({
```

Comment:
oups!! toutes les queries et mutations... Les variables en plus... là vraiment il faut cacher les mots de passe et les mails...

Reply:
after some tests, I think that the only case we could have email in variables is the creation of a user but it is done via the old api

Reply:
and passwords are marked as `[Filtered]` by sentry